Removing conda
Loading conda
Thu Feb 13 21:14:12 UTC 2025
SWOT_L2_HR_PIXC_019_548_076L_20240819T081915_20240819T081926_PIC0_01
(462798,)
type: normal
NHDPLUS_H_0102_HU4_GDB
flowlines read-in
exploded
type: normal
NHDPLUS_H_0103_HU4_GDB
flowlines read-in
exploded
type: normal
NHDPLUS_H_0104_HU4_GDB
flowlines read-in
exploded
type: normal
NHDPLUS_H_0105_HU4_GDB
flowlines read-in
exploded
INFO: Pandarallel will run on 48 workers.
INFO: Pandarallel will use Memory file system to transfer data between the main process and workers.
slurmstepd-visterra-cpu005: error: *** JOB 28915680 ON visterra-cpu005 CANCELLED AT 2025-02-13T21:14:36 ***
Traceback (most recent call last):
  File "/work/pi_cjgleason_umass_edu/.conda/envs/narrowPIXC/lib/python3.12/multiprocessing/managers.py", line 813, in _callmethod
    conn = self._tls.connection
           ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ForkAwareLocal' object has no attribute 'connection'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nas/cee-water/cjgleason/fiona/narrow_rivers_PIXC/evalCoverage.py", line 139, in <module>
